src/storage/mongoose/todo-item.ts

```diff
@@ -0,0 +1,16 @@
+import { model, Schema } from 'mongoose';
+
+const todoSchema: Schema = new Schema(
+  {
+    title: { type: String, required: true },
+  },
+  {
+    collection: 'todoItems',
+    timestamps: {
+      createdAt: 'createdAt',
+      updatedAt: 'updatedAt',
+    },
+  },
+);
+const todoItem = model('TodoItem', todoSchema);
+export default todoItem;
```

Comment:
Please do make sure you are leaving a new line by EOF for all files which are created (You can use EditorConfig - https://editorconfig.org/ which would do this automatically for you)